lib/fog/scaleway/account.rb

```diff
@@ -13,6 +13,7 @@ class APIError < Error; end
 
       requires   :scaleway_token
       recognizes :scaleway_email
+      recognizes :connection_options
```

Comment:
`connection_options` seems to be in the list of recognized options by default.

https://github.com/fog/fog-core/blob/v1.42.0/lib/fog/core/service.rb#L232-L234